release-steps.sh

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env bash
+set -eo pipefail
```

Comment:
@tomusher Does the whole file count as a pipeline?

Reply:
No, the main difference here is the inclusion of `set -e`. The `o pipefail` is just future proofing in case we do introduce some pipelines to this script.

Reply:
💡 thanks for the explanation @tomusher